examples/http-service-basic-authentication-file-user-store/http_service_basic_authentication_file_user_store.md

```diff
@@ -1,6 +1,6 @@
 # HTTP service - Basic authentication file user store
 
-The `http:Service` can be secured with basic authentication and optionally, by enforcing authorization. This can be achieved by validating the basic authentication token sent in the `Authorization` header against the provided configurations in the `Config.toml` file. This file stores the usernames and passwords for the authentication and the scopes for the authorization. An `http:Service` can configure the scopes it needs for authorization in the `auth` field of the `@http:ServiceConfig` annotation. 
+The `http:Service` can be secured with basic authentication and additionally, scopes can be added to enforce authorization. This can be achieved by validating the basic authentication token sent in the `Authorization` header against the provided configurations in the `Config.toml` file. This file stores the usernames and passwords for the authentication and the scopes for the authorization. An `http:Service` can configure the scopes it needs for authorization in the `auth` field of the `@http:ServiceConfig` annotation. 
```

Comment:
The `http:Service` can be secured with basic authentication and additionally, scopes can be added to enforce authorization. It validates the basic authentication token sent in the `Authorization` header against the provided configurations in the `Config.toml` file. The file stores the usernames and passwords for the authentication and the scopes for the authorization. To engage authentication set the default values to `fileUserStoreConfig` field and add the `Config.toml` file next to the service file. To engage authorization set scoped to `scopes` field. Both configurations must be given as part of service configuration.